package.json

```diff
@@ -30,9 +30,14 @@
     "vinyl": "^0.4.6",
     "webpack": "^1.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "gulp-jsbeautify": "^0.1.1",
+    "gulp-jscs": "^1.4.0",
```

Comment:
shouldn't we move this to the `dependencies` list instead now? since it is required in order to run the app.